src/muxer.ts

```diff
@@ -19,7 +19,7 @@ interface VideoOptions {
 	codec: typeof SUPPORTED_VIDEO_CODECS[number],
 	width: number,
 	height: number,
-	rotation?: 0 | 90 | 180 | 270
+	rotation?: 0 | 90 | 180 | 270 | RotationMatrix
```

Comment:
I mean, if we rename `RotationMatrix` to `TransformationMatrix` then it's a bit weird since the field is still called `rotation` - but I say this is still _fine_.

Changing the API now to be semantically correct would just complicate it needlessly.

Reply:
Yeah that was my reasoning for naming the type `RotationMatrix` instead of something like `TransformationMatrix` initially.  
But I think your proposed changes are a good compromise.